torch/_functorch/aot_autograd.py

```diff
@@ -2819,6 +2813,19 @@ def aot_module_simplified(
         **dict(mod.named_parameters(remove_duplicate=False)),
         **dict(mod.named_buffers(remove_duplicate=False)),
     }
+
+    tracing_context = TracingContext.get()
+    if tracing_context:
+        for name, _ in params.items():
+            name = normalize_attr_name(name)
+            tracing_context.aot_autograd_arg_pos_to_source.append(tracing_context.module_context.names_to_sources[name])
+
+        if hasattr(mod, "graph"):
```

Comment:
this is annoying but ok

Reply:
Is the thing we care about that `mod` is an instance of `torch.fx.GraphModule`?